rllib/tests/test_supported_multi_agent.py

```diff
@@ -14,7 +15,23 @@ def check_support_multiagent(alg, config):
                  lambda _: MultiAgentMountainCar({"num_agents": 2}))
     register_env("multi_agent_cartpole",
                  lambda _: MultiAgentCartPole({"num_agents": 2}))
-    config["log_level"] = "ERROR"
+
```

Comment:
Before this PR, we would use the default multiagent setup, which doesn't cover the case of having 2 policies. The default case is simply: 1 policy ("default_policy") and all agents map to this 1 policy.